src/components/programmingCard.js

```diff
@@ -0,0 +1,20 @@
+import React from "react"
+import "./programmingCard.scss"
+
+export default function ProgrammingCard({ title, summary, id }) {
+  const href = `https://www.artsmatter.org/programs/${id}`
+  return (
+    <div className="programming-card">
+      <h3 className="programming-card__title">{title}</h3>
```

Comment:
Can we change this to an h4 tag? I know it is an h3 on the existing site, but it should really be an h4 just based on how it is nested.

Reply:
Good catch, I think that makes a lot more sense too.